bin/broker/server.js

```diff
@@ -0,0 +1,71 @@
+// establish express + socket.io server
+const express = require('express');
+const socket = require('socket.io');
+const path = require('path');
+
+// port for the dashboard
+const PORT = 5001;
+
+// TODO: Get port number from metadata file
+
+// initialize express app
+const app = express();
+
+// create express server
+const server = app.listen(PORT, () => {
+    if (process.hasOwnProperty('send')) {
+        // launched as a daemon: emit success message to parent process
+        process.send({
+            status: 'success',
+            message: `\nSUCCESS: Dashboard is up at port ${PORT}.`,
+        });
+    } else if (process.env.NODE_ENV !== 'test') {
+        // launched for dev: log success to console
+        console.log(`Dashboard is running at port: ${PORT}`);
+    }
+});
+
+// error while creating express server
+server.on('error', (err) => {
+    let serverError = '\nERROR: Cannot start dashboard.';
+
+    // if port is busy, create custom error
+    if (err.code === 'EADDRINUSE') {
+        serverError += `\nPort ${PORT} is busy. 
+        Please try again on another port.`;
+    } else {
+        serverError += `\n${err.message}`;
+    }
+
+    if (process.hasOwnProperty('send')) {
+        // launched as a daemon: emit failure message to parent process
+        process.send({
+            status: 'fail',
+            message: serverError,
+        });
+    } else if (process.env.NODE_ENV !== 'test') {
```

Comment:
I think it would be better if we log to console only for "debug" environment.

Reply:
As we are now giving the option via the CLI only, for running the server as a standard process. The output is now being logged to the console.